File: Removendo de Lista - Aula 18.03/Removelista.java
import java.util.ArrayList;
import java.util.Random;
import java.util.Scanner;

public class Removelista {
    static void exibir(ArrayList<Integer> lista) {
        for (Integer item : lista) {
            System.out.println(item);
        }
    }

    public static void main (String a[]){
        
        ArrayList<Integer> lista = new ArrayList<Integer>();
        int valor, qtdNumeros=10;
        Scanner teclado = new Scanner (System.in);
        Random gerador = new Random();
        for (int i = 0; i<qtdNumeros; i++){
            valor = gerador.nextInt(100);

            if (!lista.contains(valor)) { 
                lista.add(valor);
                lista.sort(null);
            }
        }

        System.out.println("Tamanho da lista: " + lista.size());
        exibir(lista);

        System.out.println("Digite um valor para exclusão: ");
        valor = teclado.nextInt(); //int.Parse() passa de int para string

        lista.remove((Integer)valor);
        System.out.println("Tamanho da lista apos remoção: " + lista.size());
        exibir(lista);

    }
}

File: Removendo de Lista - Aula 18.03/remove_lista.py
import random

def exibir(lista):
    for item in lista: 
        print (item)

lista = []

qtd_numeros= 10
for i in range(0, qtd_numeros):
    valor = random.randint(0,100)
    if (valor not in lista):
        lista.append(valor)
        lista.sort()

exibir (lista)

valor = int(input ('entre com o valor para remocao: '))
lista.remove(valor)
print ("Tamanho da lista: ", lista.count())
exibir (lista)